frontend/src/views/Crops/utils.ts

```diff
@@ -91,19 +91,45 @@ export type CropsModalReducerAction =
   | SetDefaultYieldAction
   | RestoreDefaultYieldAction;
 
+function isCropSet(cropId: number, selectedCrop: Crop | undefined) {
+  if (cropId === 0) return false;
+  // If cropId is non-zero, selectedCrop should match it
+  if (cropId !== CROP_OTHER_ID && (selectedCrop === undefined || cropId !== selectedCrop.id)) {
+    throw new Error('Crops modal entered bad state');
+  }
+  return true;
+}
+
+function validateBushelConversion(selectedCrop: Crop | undefined) {
+  if (
+    selectedCrop === undefined ||
+    selectedCrop.harvestbushelsperton === null ||
+    selectedCrop.harvestbushelsperton === 0
+  ) {
+    throw new Error('Crops modal entered bad state');
+  }
```

Comment:
Optional: could this not just be a single `!selectedCrop?.harvestbushelperton` check for all falsy values?

Reply:
Oh, yeah, thx for catching that